src/ExtendedGridOrbitals.cc

```diff
@@ -281,6 +283,11 @@ void ExtendedGridOrbitals::initGauss(
                 xc[0] += hgrid[0];
             }
         }
+
+        MemorySpace::Memory<ORBDTYPE, memory_space_type>::copy_view_to_dev(
```

Comment:
this copy is probably incorrect since its size is the size of ncols*nrows while you try to set one column